src/routes/safe/components/Transactions/TxList/TxDetails.tsx

```diff
@@ -139,26 +147,44 @@ export const TxDetails = ({ transaction }: TxDetailsProps): ReactElement => {
         >
           <TxDataGroup txDetails={data} />
         </div>
-        {isMultiSend && getModuleDetails()}
-        <div className={cn('tx-summary', { 'will-be-replaced': willBeReplaced })}>
-          <TxSummary txDetails={data} />
-        </div>
-      </div>
-      <div>
-        {!isMultiSend && getModuleDetails()}
+      </>
+    ) : (
+      <>
         <div
-          className={cn('tx-owners', {
+          className={cn('tx-details', {
+            'no-padding': isMultiSendTxInfo(data.txInfo) || noTxDataBlock,
+            'not-executed': !data.executedAt,
             'will-be-replaced': willBeReplaced,
           })}
         >
-          <TxOwners txDetails={data} isPending={isPending} />
+          <TxDataGroup txDetails={data} />
         </div>
-        {!isPending && !data.executedAt && txLocation !== 'history' && !!currentUser && (
-          <div className={cn('tx-details-actions', { 'will-be-replaced': willBeReplaced })}>
-            <TxExpandedActions transaction={transaction} />
+        {getModuleDetails()}
+        <div className={cn('tx-summary', { 'will-be-replaced': willBeReplaced })}>
+          <TxSummary txDetails={data} />
+        </div>
+      </>
+    )
+
+  return (
+    <TxDetailsContainer ownerRows={hasModule ? 3 : 2}>
```

Comment:
How does this now interact with showing/hiding the owners? This was added to prevent a grey space below the owners section. Maybe it can now be removed?